quickjs.c

```diff
@@ -23009,7 +23009,7 @@ static JSAtom js_parse_destructuring_var(JSParseState *s, int tok, int is_arg)
    present at the top level. */
 static int js_parse_destructuring_element(JSParseState *s, int tok, int is_arg,
                                         int hasval, int has_ellipsis,
-                                        bool allow_initializer)
+                                        bool allow_initializer, bool export_flag)
```

Comment:
A suggestion for a follow-up PR: replace these 5 booleans with a flags argument. Call sites are IMO more legible as `frob(HasArg|HasVal|HasEllipsis|AllowInitializer|ExportFlag)` than `frob(true,false,false,true,true)`

(I suppose that, although its name suggests it's a boolean, has_ellipsis is actually a tri-state. The others are booleans though.)

Reply:
Good idea, will do!